Makefile

```diff
@@ -69,4 +69,7 @@ build-docs:
 
 .PHONY: push-docs
 push-docs: build-docs
-	gsutil -m rsync -a public-read -d -r ${DOCS_DEST} ${DOCS_BUCKET}
+	gsutil \
+		-h "Content-Type:text/html" \
```

Comment:
@tedmiston not every file that gets synced is html. There's also css and js, etc. Maybe just leave this line off?

Reply:
fixed